detection_rules/rule.py

```diff
@@ -591,6 +591,38 @@ def is_dirty(self) -> Optional[bool]:
         if existing_sha256 is not None:
             return existing_sha256 != self.sha256()
 
+    @property
```

Comment:
with my latest commit, these helper methods all go unused, but we can leave them for convenience

Reply:
FWIW, I think they'll be used when we reuse the 100 v bumps on new forked rules.